src/utils/status/pv/pvStatus.js

```diff
@@ -0,0 +1,30 @@
+import {
+  PV_STATUS_AVAILABLE,
+  PV_STATUS_BOUND,
+  PV_STATUS_FAILED,
+  PV_STATUS_RELEASED,
+  PV_STATUS_DEFAULT,
+} from './constants';
+
+import { getStatusPhase } from '../../../selectors';
+import { NOT_HANDLED } from '../common';
+
+const statusMapper = {
+  Available: PV_STATUS_AVAILABLE,
```

Comment:
you could convert all values to form such as `Available: { status: PV_STATUS_AVAILABLE }` and then in `getPvStatus` you could just do `pv => statusMapper[getStatusPhase(pv)] || {status: PV_STATUS_DEFAULT};`. `getMappedStatus` could be deleted then

Reply:
the statusMapper is only meant to be the mapping of constant -> constant, getMappedStatus can include other functionality in the future.

Reply:
@rawagner @suomiy Is there any action required from my side on this? I am not sure if I understood it well.

Reply:
@suomiy ok, i think there is not action needed @cloudbehl :)